propka/run.py

```diff
@@ -36,34 +36,69 @@ def main(optargs=None):
             my_molecule.write_propka()
 
 
-def single(pdbfile, optargs=None):
-    """Run a single PROPKA calculation using *pdbfile* as input.
+def single(filename: str, optargs: list = None, stream=None,
```

Comment:
Actually, I think `optargs = ()` would be the most appropriate default value (an immutable empty sequence). In similar contexts, `None` means `sys.argv`, which is not the case here.

Reply:
I don't often see empty tuples as defaults and more often see the – slightly ambiguous – `None`. But it does make very clear what the meaning of the default is so I'd be happy to use it.

Reply:
Optargs does not need to be a list, does it? A tuple would be ok, too.

Reply:
The only reason for a list would be to match 3.1's expected input, but honestly even after the changes in 46de602 you can technically still pass a list and it'll behave normally.